static/src/javascripts/projects/common/modules/analytics/beacon.js

```diff
@@ -9,20 +8,4 @@ const fire = (path) => {
     return img;
 };
 
-const postJson = (path, jsonString) => {
-    const url =
-        config
-            .get('page.beaconUrl', '')
-            .replace(/^\/\//, `${window.location.protocol}//`) + path;
-
-    fetch(url, {
-        method: 'post',
-        header: {
-            'Content-Type': 'application/json',
-        },
-        body: jsonString,
-        mode: 'cors',
-    });
-};
```

Comment:
`postJson` is not used anywhere, and a rather trivial implementation.

cc. @guardian/ophan for visibility.

Reply:
Hi, thanks for the cc- however, I don't think this code is related to Ophan - ie it's not used for writing data into Ophan's Tracker at ophan.theguardian.com - I think it's actually for monitoring ad-rendering, if I'm reading https://github.com/guardian/frontend/pull/17854/files correctly?

Reply:
Thanks, you’re right, and `performance-logging.js` was removed in #21652. Thanks @rtyley.

Reply:
Do we still need the `fire` function?

The only place its used is: [on-slot-render.js#L15](https://github.com/guardian/frontend/blob/main/static/src/javascripts/projects/commercial/modules/dfp/on-slot-render.js#L15)

Is it still useful?

Reply:
I’m proposing to remove it here: https://github.com/guardian/frontend/pull/23708